### PR TITLE
Centcom Two: It's better now, I swear.

### LINKED ISSUE #8425: The new centcom has almost no room to fight in
The first checkpoint door has centcom level access, so unless you have a borg with you to lead the charge, your stuck smacking each other in that tiny little area.

Please I gotta get my tussle on

### PR DESCRIPTION
**Pull request by Fayrik**
So there's a map change in this commit, lets start with that...
Fixes #8425. Also adds a few extra toys and whatnot to the map, along with a sorely needed coat of paint.
Z2 now looks like this, (in game screenshot):
Centcom
Next on my list, #8573 was a pretty easy fix, and here it is:
Bowman
I also took the liberty to change the Deathsquad channel to the Centcom channel. While it's always been a traditionally Deathsquad channel, the fact the ERTs use it means it's not all doom and gloom anymore. (I had planned to swap this over to an encryption key channel, but reading through the telecoms code I soon found that was a very bad idea indeed.)

Finally I took a pop at #7667. I wasn't able to fix it, but I did manage to make ERT medics spawnable in the select equipment menu, and that's frankly made my day.
... I then took another liberty of adding alt styles to the centcom uniforms.
Uniforms

### CODE REVIEW

**Comment by Cheridan** [comment]
first

**Comment by SconesC** [comment]
Do Deathsquad spawn with the CentComm Bowmans now?

**Comment by Fayrik** [comment]
@SconesC: Yep. Deathsquads and ERTs get the bowmans, everything else centcom-related now gets the standard. So yeah. Deathsquads will now be totally unstoppable against virtually anything the crew has... Again.
Although, if I'm totally honest, the only reason I wanted to add it is because I wanted one of those rad bowman sprites for muh snowflake.

**Comment by Steelpoint** [comment]
:+1:

File: config/admins.txt [hunk 1]
```diff
@@ -79,4 +79,5 @@ dannno = Game Master
 allura = Game Master
 lo6a4evskiy = Game Master
 vekter = Game Master
-Ahammer18 = Game Master
\ No newline at end of file
+Ahammer18 = Game Master
+Fayrik = Game Master
```
Comment (Fayrik) [inline config/admins.txt]: Oh, also adds me as an admin in admins.txt.
Ahhh accidental stealth changes!

**Comment by Lumipharon** [comment]
Speaking of unstoppable death squad... why was the ammo count on the pulse rifle nerfed, hard?
For an admin only squad of maximum rape, every time you see them, they run out after 60 seconds of fighting, where as before is was like, 200 shots or something.

**Comment by Steelpoint** [comment]
Yeah it was around 200 rounds prior, paprika for some reason heavily nerfed the Pulse Rifles for some reason.

File: config/admins.txt (hunk 1 above)
Comment (Miauw62) [inline config/admins.txt]: *conspiracy intensifies*

**Comment by Fayrik** [comment]
I can confirm, the pulse weapon capacities are now 10, 25 and 40. (That's Pistol, Carbine and Rifle.)
I think I'm going to open a forum feedback thread on the matter, collect ideas there and make a second, rebalance-oriented PR for that once this one's done... Mostly because I don't want to have to ever have to deal with merge conflicts on a map file ever again. :sob:

**Comment by Niknakflak** [comment]
I dont really approve of the weapons at the end of the round because muh centcom consistency  and the traitor weapons are out of place but that doesnt matter too much.

However, the last time you did this map change, it only applied to the box station map.  Can you get it to apply to both centcoms because they dont have any differences between the two maps.

**Comment by Fayrik** [comment]
@Niknakflak: Currently, those are toys. I just needed something to fill the space.

**Comment by Fayrik** [comment]
New commit... And...
Fixes #7667. Everything mentioned in that issue report is **fixed**.
 * Medical ERT members now spawn from the equipment menu.
 * All 7 ERT members spawn geared.
 * You can now spawn multiple ERTs or Deathsquads.

Other things include:
 * Deathsquad members now display their loyalty implant. (They were there before, just hidden.)
 * Deathsquad members now start with loyalty pinned pulse rifles.
 * Pulse rifles now have 80 shots, not 40. Not nearly 200, but lets see how this change goes down first. Baby steps.
 * Admins can no longer runtime the server by trying to spawn traitors in a round that hasn't started.

**Pull request merged**